myScript.js

```diff
@@ -39,4 +39,149 @@ function createList() {
   };
 }
 
-document.getElementById('humberger-icon').onclick = createList; // modify
+document.getElementById('humberger-icon').onclick = createList;
+
+/// /////////////////////////////////////////////////////////////////////
+
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
```

Comment:
- Please on your popup design, kindly try to stick to the Figma design as possible, it will help in your attention to detail skill. Please work on the following so it same as the Figma design template.

  - The padding of the buttons is to be similar to that of the Figma details,  
  - Font size and font family should be the same as the Figma design
  -  [Optional] Add a `cursor: pointer` CSS property on the button class and the `x` for closing the popup, this will improve the user experience.
  - [Optional] **Recommended** PLease remove the yellow colored border around the popup card, its mot part of the design.

##### Your design 
![Screenshot from 2022-07-26 20-44-46](https://user-images.githubusercontent.com/58771507/181104187-83a4bad4-2a38-4b0e-b367-c061559ef7e5.png)


##### Figma design 

![Screenshot from 2022-06-24 17-30-55](https://user-images.githubusercontent.com/58771507/181104233-1a7add4c-5e3a-413d-8b11-1d69d63c89d7.png)